lib/reporting/fraud_metrics_lg99_report.rb

```diff
@@ -121,8 +128,53 @@ def cloudwatch_client
     end
 
     def lg99_unique_users_count
-      @lg99_unique_users_count ||=
-        (data[Events::IDV_PLEASE_CALL_VISITED] + data[Events::IDV_SETUP_ERROR_VISITED]).count
+      @lg99_unique_users_count ||= (data[Events::IDV_PLEASE_CALL_VISITED] +
+        data[Events::IDV_SETUP_ERROR_VISITED]).count
+    end
+
+    def unique_suspended_users_count
+      @unique_suspended_users_count ||= data[Events::SUSPENDED_USERS].count
+    end
+
+    def user_days_to_suspension_avg
+      user_data = User.where(uuid: data[Events::SUSPENDED_USERS]).pluck(:created_at, :suspended_at)
+      return 'n/a' if user_data.empty?
+
+      difference = user_data.map { |created_at, suspended_at| suspended_at - created_at }
+      convert_to_days(difference.sum / difference.size)
+    end
+
+    def user_days_proofed_to_suspension_avg
+      user_data = User.joins(:profiles).group('users.id').pluck(
+        'MAX(profiles.verified_at)',
+        :suspended_at,
+      )
+
+      return 'n/a' if user_data.empty?
+
+      difference = user_data.map { |profiled_at, suspended_at| suspended_at - profiled_at }
+      convert_to_days(difference.sum / difference.size)
+    end
+
+    def unique_reinstated_users_count
+      @unique_reinstated_users_count ||= data[Events::REINSTATED_USERS].count
+    end
+
+    def user_days_to_reinstatement_avg
+      user_data = User.where(uuid: data[Events::REINSTATED_USERS]).pluck(
+        :suspended_at,
+        :reinstated_at,
+      )
+      return 'n/a' if user_data.empty?
+
+      difference = user_data.map { |suspended_at, reinstated_at| reinstated_at - suspended_at }
+      convert_to_days(difference.sum / difference.size)
+    end
+
+    private
+
+    def convert_to_days(seconds)
+      (seconds / (24 * 3600)).round(2)
```

Comment:
We can use Rails here
```suggestion
      seconds.seconds.in_days.round(2)
```

Reply:
yeah, I may just forgo this method and inline the in_days